tests/integration/test_charm.py

```diff
@@ -143,8 +148,8 @@ async def test_configmap_contents_with_relations(
     ops_test: OpsTest, copy_libraries_into_tester_charm, lightkube_client: Client
 ):
     """Tests the contents of the dashboard sidebar link configmap when relations are present."""
-    tester1 = "kubeflow-dashboard-requirer-tester1"
-    tester2 = "kubeflow-dashboard-requirer-tester2"
+    tester1 = f"{TESTER_CHARM_NAME}1"
+    tester2 = f"{TESTER_CHARM_NAME}2"
     charm = await ops_test.build_charm("./tests/integration/sidebar_requirer_tester_charm")
```

Comment:
Can you make it clear in the docstring of  the test that we're using this test charm and what it does (mock charm?), I had no idea what it is and went to look at the files there

Reply:
added!